sql_driver/lib.go

```diff
@@ -16,7 +19,17 @@ func (d *LibsqlDriver) Open(dbPath string) (driver.Conn, error) {
 	if strings.HasPrefix(dbPath, "file:") {
 		return (&sqlite3.SQLiteDriver{}).Open(dbPath)
 	}
-	return sqlddriver.SqldConnect(dbPath), nil
+	if strings.HasPrefix(dbPath, "wss://") {
```

Comment:
We should parse the URL properly, instead of matching on substrings.
https://pkg.go.dev/net/url

Reply:
Also, we should allow `ws` for WebSocket without TLS (and `http` for HTTP without TLS), for example if you want to target a local `sqld` instance.

Reply:
I think it's an overkill to parse the whole url just to check the scheme when this can easily be done with the prefix but I will do it for you if it makes you feel better.

Good point with the `ws` and `http` though.

Reply:
done.